### PR TITLE
Update README.md with GH workflow link

### PR DESCRIPTION
**Pull request by danielpeintner**

### CODE REVIEW

**Comment by salmonb** [comment]
Thank you for spotting the broken link 👍

Actually the correct link is https://github.com/webfx-demos/webfx-demo-fx2048/blob/webfx/.github/workflows/builds.yml (the master branch is the original game, the webfx port is in the webfx branch).

Can you please update it so I can merge this PR?

File: .github/README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ WebFX doesn't target only the Web. WebFX applications can be compiled to run nat
 
 
 You can check out the demos to see how a GitHub workflow can generate these executables.
-For example, here is the [GitHub workflow](https://github.com/webfx-demos/webfx-demo-fx2048/blob/main/.github/workflows/builds.yml) for the FX2048 demo and the [executables](https://github.com/webfx-demos/webfx-demo-fx2048/releases) that it generated.
+For example, here is the [GitHub workflow](https://github.com/webfx-demos/webfx-demo-fx2048/blob/master/.github/workflows/gradle.yml) for the FX2048 demo and the [executables](https://github.com/webfx-demos/webfx-demo-fx2048/releases) that it generated.
```
Comment (danielpeintner) [inline .github/README.md:106]:
```suggestion
For example, here is the [GitHub workflow](https://github.com/webfx-demos/webfx-demo-fx2048/blob/webfx/.github/workflows/builds.yml) for the FX2048 demo and the [executables](https://github.com/webfx-demos/webfx-demo-fx2048/releases) that it generated.
```

**Comment by danielpeintner** [comment]
> Can you please update it so I can merge this PR?

Done so with https://github.com/webfx-project/webfx/pull/31/commits/a0fca8d324fb40b4cb958d2075fc18deae8ead44

**Comment by salmonb** [comment]
Ok, merged, thank you!

**Pull request merged**